### ISSUE TITLE
Warm-up failed | dynamo

### ISSUE THREAD

**Issue by lbaliunas** [davidbrowne17/csm-streaming#8]
Hi, thanks for the repo!

Do you maybe know what maybe causing this error during warm-up?

`Warm-up failed, but continuing anyway: call_method NNModuleVariable() _forward_unimplemented [LazyVariableTracker(), LazyVariableTracker(), LazyVariableTracker()] {}

from user code:
   File "/workspace/.venv/lib/python3.10/site-packages/torch/_dynamo/external_utils.py", line 38, in inner
    return fn(*args, **kwargs)
  File "/workspace/.venv/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1562, in _call_impl
    return forward_call(*args, **kwargs)`

Package versions:
Torch version: 2.4.0+cu124
Torchaudio version: 2.4.0+cu124
Torch CUDA available: True
Torch CUDA version: 12.4

**Comment by miku448** [comment]
I don't know if it's relevant, but I had a similar issue during warm-up and had too add 
```python
import torch._dynamo
torch._dynamo.config.suppress_errors = True
```
to the `main.py` file to make it work.

Additionally, some caveats:
* Had to change `ws://` to `wss://` in `app.js` because I was using an https server.
* `Llama-3.2-1B` is too unstable for me. `meta-llama/Llama-3.2-3B-Instruct` works a lot better and it fits in a 3090 along with the other models.
* Sometimes the audio crops in the middle and doesn't work well. I will try with another reference. I can't tell if it's a problem with the sesame model or the audio streaming yet.

**Comment by davidbrowne17** [comment]
Are you using the base or a fine-tuned version? The base model is quite bad but sounds good when fine-tuned.
